vllm/core/block/block_table.py

```diff
@@ -89,7 +91,8 @@ def allocate(self,
 
     def append_token_ids(self,
                          token_ids: List[int],
-                         num_lookahead_slots: int = 0) -> None:
+                         num_lookahead_slots: int = 0,
+                         num_computed_slots: Optional[int] = None) -> None:
```

Comment:
nit: num_computed_tokens?

Reply:
well, we have `num_full_slots`; it's really the slots (or rather the data in them) that's being computed, not the tokens

Reply:
hmm but you are passing num_computed_tokens to this API? What's the definition of slots here?

Reply:
I would say it's the same thing, it's just strange that they are named "computed tokens" at the call site - it's the KV entries being computed, not tokens (tokens are known beforehand)

Reply:
but happy to change if you feel strongly about it!

Reply:
oh I was the one who decided that name lol. I think it was because 1 token == 1 kv. I am fine with keeping it slots (or renaming other name). But feel like it is better being consistent